it/controllers/CalculationControllerISpec.scala

```diff
@@ -517,16 +805,82 @@ class CalculationControllerISpec extends IntegrationSpecCommonBase {
+        returnPenaltyDetailsStub(penaltyDetailsWithAdditionalDuePenalty)
+        val request = controller.onPageLoad("65431234567890", true)(fakeRequest)
+        status(request) shouldBe Status.OK
+        val parsedBody = Jsoup.parse(contentAsString(request))
+        parsedBody.select("#main-content h1").first().ownText() shouldBe "Additional penalty"
+        parsedBody.select("#main-content header p .govuk-visually-hidden").first.text() shouldBe "The period dates are"
+        parsedBody.select("#main-content header p").first.text() shouldBe "The period dates are 1 January 2021 to 1 February 2021"
+        parsedBody.select("#main-content p").get(1).text() shouldBe
+          "The additional penalty is charged from 31 days after the payment due date, until the total is paid."
+        parsedBody.select("#main-content .govuk-summary-list__row").get(0).select("dt").text() shouldBe "Penalty amount (estimate)"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(0).select("dd").text() shouldBe "£123.45"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(1).select("dt").text() shouldBe "Number of days since day 31"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(1).select("dd").text() shouldBe "9 days"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(2).select("dt").text() shouldBe "Additional penalty rate"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(2).select("dd").text() shouldBe "4%"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(3).select("dt").text() shouldBe "Calculation"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(3).select("dd").text() shouldBe "VAT amount unpaid × 4% × number of days since day 31 ÷ 365"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(4).select("dt").text() shouldBe "Amount received"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(4).select("dd").text() shouldBe "£113.45"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(5).select("dt").text() shouldBe "Amount left to pay"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(5).select("dd").text() shouldBe "£10.00"
+        parsedBody.select("#main-content p").get(2).text() shouldBe
+          "Penalties and interest will show as estimates if HMRC does not have enough information to calculate the final amounts."
+        parsedBody.select("#main-content a").attr("href") shouldBe "/penalties"
+      }
+
+      "return 500 (ISE) when the user specifies a penalty not within their data" in new Setup (true) {
+        returnPenaltyDetailsStub(samplePenaltyDetails)
+
+        val request = controller.onPageLoad("123456800", true)(fakeRequest)
+        status(request) shouldBe Status.INTERNAL_SERVER_ERROR
+      }
+
+      "return 300 (SEE_OTHER) when the user is not authorised" in new Setup (true) {
```

Comment:
Could you name the parameter in the Setup class and change status to 303
```suggestion
      "return 303 (SEE_OTHER) when the user is not authorised" in new Setup (true) {
```